### PR TITLE
added nav bar and header

### PR DESCRIPTION
**Pull request by coder-mezab**
created nav bar and header

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -3,3 +3,78 @@
...
+}
+.navbar {
+  display: flex;
+  padding: 20px;
+  align-items: center;
+  background: -webkit-radial-gradient(rgb(158, 255, 214), #55cfff);
+  height: 75px;
+}
+nav {
+  flex: 2;
```
Comment (faran4engg) [inline css/style.css:20]: can you explain what does `flex: 2` do?

File: css/style.css [hunk 2]
```diff
@@ -3,3 +3,78 @@
...
+  background-color: #eee;
+}
+.row {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: auto;
+  justify-content: space-around;
+  background: -webkit-radial-gradient(#fff, #c9c506);
+  position: relative;
```
Comment (faran4engg) [inline css/style.css:49]: I would arrange properties in the correct group. for example:

```css
  display: flex;
  justify-content: space-around;
  align-items: center;
  flex-wrap: wrap;
  
  position: relative;
  margin: auto;
  background: -webkit-radial-gradient(#fff, #c9c506);
```

it helps in reading code @coder-mezab

File: index.html [hunk 3]
```diff
@@ -4,7 +4,46 @@
...
+        </p>
+        <a href="" class="btn"> Explore Now &#8594; </a>
+      </div>
+      <div class="col-1">
+        <img src="css/images/slogo.png" alt="shoes image" />
+      </div>
+    </div>
+  </div>
+
   <body></body>
```
Comment (faran4engg) [inline index.html:48]: I think everything from line 16 to 47 should be inside `body` :)

File: css/style.css [hunk 4]
```diff
@@ -3,3 +3,78 @@
   margin: 0;
   box-sizing: border-box;
 }
+body {
+  font-family: "Poppins", sans-serif;
```
Comment (faran4engg) [inline css/style.css:7]: I think, with current html code outside body, this font family wont be applied :)

File: css/style.css (hunk 1 above)
Comment (coder-mezab) [reply css/style.css:20]: its the flex grow rate it will grow double in comparison of others @faran4engg

File: css/style.css (hunk 2 above)
Comment (coder-mezab) [reply css/style.css:49]: I will take that into consideration

File: css/style.css (hunk 4 above)
Comment (coder-mezab) [reply css/style.css:7]: i will wornk on that issue @faran4engg

File: css/style.css [hunk 5]
```diff
@@ -3,3 +3,78 @@
...
+  align-items: center;
+  background: -webkit-radial-gradient(rgb(158, 255, 214), #55cfff);
+  height: 75px;
+}
+nav {
+  flex: 2;
+  text-align: right;
+}
+nav ul {
+  list-style-type: none;
```
Comment (fayez-baig) [inline css/style.css:24]: can you tell me what is the shorthand property for `list-style-type`

**Review by fayez-baig — COMMENTED** [review]
check comments

File: css/style.css (hunk 5 above)
Comment (coder-mezab) [reply css/style.css:24]: I haven't learned it yet @fayez-baig

Comment (fayez-baig) [reply css/style.css:24]: Google it bro.

File: css/style.css (hunk 4 above)
Comment (coder-mezab) [reply css/style.css:7]: @faran4engg i have fixed the issue

**Review by faran4engg — APPROVED** [review]
great 1st PR. LGTM. 
approved 💪🏼

File: css/style.css (hunk 5 above)
Comment (faran4engg) [reply css/style.css:24]: @coder-mezab but answer this before merging :)

Comment (coder-mezab) [reply css/style.css:24]: list-style is the short hand property of list style type

**Review by fayez-baig — APPROVED** [review]

File: css/style.css (hunk 2 above)
Comment (coder-mezab) [reply css/style.css:50]: i have arranged it accordingly @faran4engg

**Review by fayez-baig — APPROVED** [review]
LGTM...you can now merge @coder-mezab 👍

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]

**Pull request merged**